### PR TITLE
Document building and debugging with XCode.

### PR DESCRIPTION
**Pull request by sethml**
I forget every time, so this documentation is as much for me as anybody else. :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +68,14 @@ Orca Slicer's logo is designed by community member Justin Levine(@freejstnalxndr
   - Tools needed: Xcode, Cmake, git, gettext, libtool, automake, autoconf, texinfo
       - You can install most of them by running `brew install cmake gettext libtool automake autoconf texinfo`
   - run `build_release_macos.sh`
+  - To build and debug in XCode:
+      - run `XCode.app`
+      - open `build_arm64/OrcaSlicer.xcodeproj`
```
Comment (Noisyfox) [inline README.md:73]: Maybe `build_<arch>` just for us folks still using intel macs 😺

**Review by Noisyfox — DISMISSED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (sethml) [reply README.md]: Done!

**Review by Noisyfox — APPROVED** [review]

**Comment by Noisyfox** [comment]
Looking good! Thanks!

**Pull request merged**